packages/greetings-english/0.1.0/_manifest.yml

```diff
@@ -1,7 +1,7 @@
 author: Katrin Leinweber
 description: A few greeting snippets
-homepage: https://github.com/katrinleinweber/espanso-greetings-english
+homepage: https://github.com/espanso/hub/tree/main/packages/greetings-english/0.1.0
```

Comment:
Would you kindly update the package paths and [`_manifest.yml`](https://espanso.org/docs/packages/package-specification/#the-_manifestyml-file) versions to `0.1.1` or `0.2.0` please, as this is an update?